### ISSUE TITLE
About ChatGPT's answer

### ISSUE THREAD

**Issue by pinkyshit** [assafdori/bypass-mdm#8]
Forgive me that I don't know coding. I asked ChatGPT that if MDM can be totally gotten rid of , its answer is no. Any mean of circumvent is temporary. Cuz even the MDM profile can be deleted, the serial number is till tied to apple's DEP(device enrollment program)which will periodically check the status of device when it connects to the internet. And probably, on some time, MDM will be enforced. So, we can fool
the organization,but the apple company. Is ChatGPT right? Temporarily?

**Comment by pinkyshit** [comment]
Ok, thanks for the reply.Can I understand it this way? Resetting to the factory mode is not recommended, unless it is inevitable.even  the bypass can be made after resetting based on the instructions you provided, but it is still an annoying ado and facing more uncertainties.

**Comment by assafdori** [comment]
> Ok, thanks for the reply.Can I understand it this way? Resetting to the factory mode is not recommended, unless it is inevitable.even the bypass can be made after resetting based on the instructions you provided, but it is still an annoying ado and facing more uncertainties.

Correct. Factory resetting will simply lead to having to do the bypass all over again, which is an inconvenience.

**Comment by mateussiqueira** [comment]
ChatGPT answers about MDM bypass can be unreliable since the landscape changes with every macOS release. The scripts in this repo are maintained and tested by the community across multiple macOS versions. Always prefer the actual scripts over AI-generated commands.
